### DISCUSSION TITLE
前端应用切换为什么有时候有弹窗,有时候没有

### DISCUSSION THREAD

**Discussion by IReigns** [zuihou/zuihou-admin-cloud#352] [02.提问]
如题,请看图片,为什么有的时候他在左上角可以无缝切换,有时候有弹窗,都是用的最新前端
无缝切换很丝滑,但是有弹窗的要重新刷新整个页面,速度很慢
![image](https://github.com/user-attachments/assets/d722457e-f130-449d-9599-a1a4e45e32d2)

![image](https://github.com/user-attachments/assets/ac1cf282-c74a-4679-b332-7db2dee19107)

**Comment by zuihou** [comment]
<img width="508" alt="image" src="https://github.com/user-attachments/assets/92f584af-7734-4b02-8e81-b933cab3ce19">

这里可以修改
